### PR TITLE
some message

### PR DESCRIPTION
**Pull request by perronogrande**
- [DEMO LINK](https://perronogrande.github.io/layout_catalog/)
- [TEST REPORT LINK](https://perronogrande.github.io/layout_catalog/report/html_report/)

A MEESAGE TO MENTOR:
so I added div to an active link in nav because it kept telling me that a tag cannot have 2 classes. it's weird, because in other tasks there wasn't such a problem. also, it doesn't pass the tests because of minor differences that to be honest would take a lot of time to find a fix. it's graphically the same though.

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,223 @@
...
+  left: 0;
+}
+
+.content {
+  display: flex;
+  width: 100%;
+}
+
+.card {
+
```
Comment (AndreyMalyshev147) [inline src/styles/main.scss:87]: Лишний отступ
```suggestion
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,223 @@
...
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 406px;
+  justify-content: center;
+  margin-top: 50px;
+  padding: 0 16px;
+
+  &s {
```
Comment (AndreyMalyshev147) [inline src/styles/main.scss:96]: Что это за класс такой?)

**Review by AndreyMalyshev147 — CHANGES_REQUESTED** [review]
Убери отступ и расскажи мне лично, что это за класс)
Ошибкой посчитать не могу, поскольку интересно)

File: src/styles/main.scss (hunk 2 above)
Comment (perronogrande) [reply src/styles/main.scss:96]: это типо cards)). есть card, а есть cards, который их обьеденяет

**Review by AndreyMalyshev147 — APPROVED** [review]
Great work!

File: src/index.html [hunk 3]
```diff
@@ -7,6 +7,326 @@
...
+          alt="Logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <div class="is-active">
+              <a class="text-link" href="">APPLE</a>
+            </div>
+          </li>
```
Comment (aliceblunt8) [inline src/index.html:24]: > add class is-active to the first link (Apple) in navigation
```suggestion
          <li class="nav__item">
              <a class="nav__link is-active" href="">APPLE</a>
          </li>
```

File: src/index.html [hunk 4]
```diff
@@ -7,6 +7,326 @@
...
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <a class="card__text">
+              Отзывов:&nbsp;5
+            </a>
+          </div>
+          <div class="price">
+            <h3 class="price__title">Цена:</h3>
+            <p class="price__value">69 999 грн</p>
```
Comment (aliceblunt8) [inline src/index.html:87]:
```suggestion
          <div class="card__price">
            <h3 class="card__price-title">Цена:</h3>
            <p class="card__price-value">69 999 грн</p>
```

File: src/index.html [hunk 5]
```diff
@@ -7,6 +7,326 @@
...
+          </div>
+          <button data-qa="card-hover" class="button">КУПИТЬ</button>
```
Comment (aliceblunt8) [inline src/index.html:89]:
```suggestion
          <button data-qa="card-hover" class="card__button">
            КУПИТЬ
          </button>
```

File: src/index.html [hunk 6]
```diff
@@ -7,6 +7,326 @@
...
+        </div>
+        <div class="card" data-qa="card">
```
Comment (aliceblunt8) [inline src/index.html:227]: attribute data-qa="card" should be added only to the first card

File: src/styles/main.scss (hunk 2 above)
Comment (aliceblunt8) [reply src/styles/main.scss:95]: we don't do such thing in scss, move cards to the separate class above card itself

File: src/styles/main.scss [hunk 7]
```diff
@@ -1,3 +1,222 @@
...
+.stars--1 > .stars__star:nth-child(-n+1),
+.stars--2 > .stars__star:nth-child(-n+2),
+.stars--3 > .stars__star:nth-child(-n+3),
+.stars--4 > .stars__star:nth-child(-n+4),
+.stars--5 > .stars__star:nth-child(-n+5) {
+  background-image: url("../images/star-active.svg");
+}
+
+.price {
+  font-family: Roboto, sans-serif;
```
Comment (aliceblunt8) [inline src/styles/main.scss:176]: remove, you've already added to html
```suggestion
```

File: src/styles/main.scss [hunk 8]
```diff
@@ -1,3 +1,222 @@
...
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  align-items: center;
+  margin-bottom: 16px;
+
+  &__title {
+    font-family: Roboto, sans-serif;
```
Comment (aliceblunt8) [inline src/styles/main.scss:184]: same

File: src/styles/main.scss [hunk 9]
```diff
@@ -1,3 +1,222 @@
...
+    font-style: normal;
+    font-weight: normal;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__value {
+    font-family: Roboto, sans-serif;
```
Comment (aliceblunt8) [inline src/styles/main.scss:193]: same

File: src/styles/main.scss [hunk 10]
```diff
@@ -1,3 +1,222 @@
...
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+}
+
+.button {
+  font-family: Roboto, sans-serif;
```
Comment (aliceblunt8) [inline src/styles/main.scss:203]: same

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**